### PR TITLE
docs(configuration): Minor grammatical fixes

### PR DESCRIPTION
**Pull request by jamesgeorge007**
Follow up of #3117 #2785

### CODE REVIEW

File: src/content/configuration/dev-server.md [hunk 1]
```diff
@@ -929,7 +932,7 @@ webpack-dev-server --pfx /path/to/file.pfx
 
 `string`
 
-The passphrase to a SSL PFX file.
+The passphrase to an SSL PFX file.
```
Comment (montogeek) [inline src/content/configuration/dev-server.md]: No sure about this one https://english.stackexchange.com/questions/1016/do-you-use-a-or-an-before-acronyms-initialisms

Comment (jamesgeorge007) [reply src/content/configuration/dev-server.md]: It is subjective. I would go for the `an` version.

**Comment by jamesgeorge007** [comment]
\cc @montogeek

**Comment by jamesgeorge007** [comment]
@wizardofhogwarts the respective PR's were there for a long time with no further action made to resolve the conflicts. And don't worry, you're gonna get the commit attribution :)

File: src/content/configuration/dev-server.md (hunk 1 above)
Comment (EugeneHlushko) [reply src/content/configuration/dev-server.md]: lets revert as its debatable please

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**